### PR TITLE
shadow branding

### PR DESCRIPTION
**Pull request by ugocupcic**

### CODE REVIEW

**Comment by ugocupcic** [comment]
Lots of auto generated noise (when building)

File: generators/python/grasp.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+Blockly.Python['grasp'] = function(block) {
+  var text_name = block.getFieldValue('NAME');
+  var value_name = Blockly.Python.valueToCode(block, 'NAME', Blockly.Python.ORDER_ATOMIC);
+  // TODO: Assemble Python into code variable.
+  var code = ""
```
Comment (ugocupcic) [inline generators/python/grasp.js:6]: @kirstyellis this is where our code would go

File: msg/json/en.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
 	"@metadata": {
 		"author": "Ellen Spertus <ellen.spertus@gmail.com>",
```
Comment (kirstyellis) [inline msg/json/en.json:3]: do we want to change the author here?

Comment (ugocupcic) [inline msg/json/en.json:3]: She's the one who did blockly so probably not :)

**Pull request merged**